main.tf

```diff
@@ -40,5 +40,52 @@ resource "azurerm_dns_a_record" "wowza" {
   resource_group_name = var.dns_resource_group
   ttl                 = 300
   records             = [var.lb_IPaddress]
-  tags                = local.common_tags
-}
\ No newline at end of file
+
+  tags = local.common_tags
+}
+
+# =================================================================
+# =================    automation account    ======================
+# =================================================================
+resource "azurerm_automation_account" "vm-start-stop" {
+  count = var.azdo_pipe_to_change_vm_status == true ? 1 : 0
```

Comment:
Do we need a flag here or can we creating for all environments then change the schedule for production to just not run.

Reply:
I thought a flag was desirable as it was easier to control the creation of the AA. However, if I am moving the AA  (see prev. comment) into a for each loop, then this can be handled there without use of a flag.

Reply:
flag removed.